rtmp-client/src/main/java/net/butterflytv/rtmp_client/RtmpClient.java

```diff
@@ -168,10 +398,14 @@ public int write(byte[] data, int offset, int size) throws IOException {
      *
      * @return true if it is successfull else returns false
      */
-    public boolean pause(boolean pause) {
-        return nativePause(pause, rtmpPointer);
+    public boolean pause(boolean pause) throws RtmpIOException, IllegalStateException {
+        int ret = nativePause(pause, rtmpPointer);
+        if (ret < RtmpIOException.RTMP_ERROR_NONE) {
```

Comment:
nit - Use `!=` instead of `<`, so the reader of the code doesn't have to understand all error codes are smaller than `RTMP_ERROR_NONE` to understand the code.

Note: This probably requires removal of `RTMP_SUCCESS`.

Reply:
Done.